src/sentry/api/serializers/rest_framework/util_json.py

```diff
@@ -0,0 +1,39 @@
+"""
+Wrapper for the builtin json module that ensures compliance with the JSON spec.
+REST framework should always import this wrapper module in order to maintain
+spec-compliant encoding/decoding. Support for non-standard features should be
+handled by users at the renderer and parser layer.
+"""
+
+from __future__ import absolute_import
+
+import functools
+import json  # noqa
+
+
+def strict_constant(o):
+    raise ValueError('Out of range float values are not JSON compliant: ' + repr(o))
+
+
+@functools.wraps(json.dump)
+def dump(*args, **kwargs):
+    kwargs.setdefault('allow_nan', False)
```

Comment:
We already have sentry.utils.json do we need more json handling utilities for this field?